wdl2cwl/tests/cwl_files/vardict.cwl

```diff
@@ -10,7 +10,7 @@ requirements:
             set -e -o pipefail
             export JAVA_OPTS="-Xmx$(inputs.javaXmx) -XX:ParallelGCThreads=1"
             vardict-java \
-            -th  $(inputs.threads) \
+            $("-th " + inputs.threads) \
```

Comment:
The other values below are in a different format, `-PARAM $(expr)`, rather than this one `("-PARAM" + expr)`. Is there are reason for one format over the other?

Reply:
Good question. This is the same pattern in the original WDL

https://github.com/common-workflow-lab/wdl-cwl-translator/blob/97a24489bb57804a659dabba38ef4bd64949293a/wdl2cwl/tests/wdl_files/vardict.wdl#L62